macro/run_simScript.py

```diff
@@ -2,6 +2,7 @@
 import os
 import sys
 import ROOT
+ROOT.gSystem.Load('libEGPythia6')
```

Comment:
Where is pythia6 used in run_simScript.py?

Reply:
Auto-load should work fine, see #633

Reply:
If you didn't explicitly check that this is needed, I would remove this line until someone shows that it is needed.